ros/install.yaml

```diff
@@ -9,3 +9,11 @@
 
 - type: target-now
   name: catkin
+
+# Symlink python to python3 as ROS depends on it:
+# https://wiki.ros.org/UsingPython3/SourceCodeChanges
+- type: system
+  default:
+    name: python-is-python3
+  bionic:
+    name: null
```

Comment:
We have included this in `tue-env` on noetic-devel

Reply:
It's ROS specific, the environment doesn't care?

Reply:
We have included  `python-is-python3` in `tue-env`. So we don't have to care about this anywhere else anymore.